### ISSUE TITLE
DMA for l552

### ISSUE THREAD

**Issue by tlatzko** [David-OConnor/stm32-hal#104]
Hi, 
are there any plans to implement the DMA for l552? I was not able to find any working DMA solution. 
Best
Thomas

**Comment by David-OConnor** [comment]
No. I think I tried, but ran into PAC-related issues. Probably not a huge lift, but I don't have time ATM. PRs accepted.
